skipper.go

```diff
@@ -985,7 +990,16 @@ func listen(o *Options, mtr metrics.Metrics) (net.Listener, error) {
 	}
 
 	if !o.EnableTCPQueue {
-		return net.Listen("tcp", o.Address)
+		ln, err := net.Listen("tcp", o.Address)
+		if o.ProxyProtocolEnable {
+			log.Printf("Proxy protocol listener")
+			pl := &proxyproto.Listener{
+				Listener:          ln,
+				ReadHeaderTimeout: 10 * time.Second,
+			}
+			return pl, err
```

Comment:
Same thing.
I'm not sure how it will be taken by original team of `zalando/skipper`. But IMO basically you don't need to allocate new variable. You can allocate your `proxyproto.Listener` to existing variable and pass current `net.Listener` as an internal `Listener` to the `proxyproto.Listener`.
```suggestion
			ln = &proxyproto.Listener{
				Listener:          ln,
				ReadHeaderTimeout: 10 * time.Second,
			}
```